index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rizwan hasan portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+  </head>
+
+  <body>
+
+    <nav>
+      <div class="navbar">
+        <h1 class="logo"> My Logo </h1>
+
```

Comment:
Kindly add the hamburger button to your design. You could follow Figma's as a guideline to be sure you are following the appropriate components 🤓

![image](https://user-images.githubusercontent.com/15807343/148621877-ba447c37-c6f1-4466-a337-d6447d11e5ad.png)

Your design
![image](https://user-images.githubusercontent.com/15807343/148621898-7ac524c2-1f6c-46c2-9e8b-cbd5b7b2d0fb.png)

Reply:
overlapping has been resolved, margin has been added to prevent this from happening again

Reply:
Addressed 🔥